gobblin-runtime/src/main/java/gobblin/runtime/instance/DefaultGobblinInstanceDriverImpl.java

```diff
@@ -137,7 +137,8 @@ public JobSpecRunnable(JobSpec jobSpec) {
 
     @Override
     public void run() {
```

Comment:
Run is generally not used in an `AbstractIdleService`, should you use a `AbstractExecutionThreadService` instead?